src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -41,21 +42,29 @@ def load_arguments(self, _):
         c.argument('service_runtime_subnet', options_list=['--service-runtime-subnet', '--svc-subnet'], help='The name or ID of an existing subnet in "vnet" into which to deploy the Spring Cloud service runtime. Required when deploying into a Virtual Network.', validator=validate_vnet)
         c.argument('service_runtime_network_resource_group', options_list=['--service-runtime-network-resource-group', '--svc-nrg'], help='The resource group where all network resources for Azure Spring Cloud service runtime will be created in.', validator=validate_node_resource_group)
         c.argument('app_network_resource_group', options_list=['--app-network-resource-group', '--app-nrg'], help='The resource group where all network resources for apps will be created in.', validator=validate_node_resource_group)
-
     with self.argument_context('spring-cloud update') as c:
         c.argument('sku', type=str, validator=validate_sku, help='Name of SKU, the value is "Basic" or "Standard"')
 
     for scope in ['spring-cloud create', 'spring-cloud update']:
         with self.argument_context(scope) as c:
             c.argument('app_insights_key',
-                       help="Instrumentation key of the existing Application Insights to be added for the distributed tracing",
+                       help="Instrumentation key of the existing Application Insights.",
                        validator=validate_tracing_parameters)
             c.argument('app_insights',
                        help="Name of the existing Application Insights in the same Resource Group. Or Resource ID of the existing Application Insights in a different Resource Group.",
                        validator=validate_tracing_parameters)
             c.argument('disable_distributed_tracing',
                        arg_type=get_three_state_flag(),
                        help="Disable distributed tracing, if not disabled and no existing Application Insights specified with --app-insights-key or --app-insights, will create a new Application Insights instance in the same resource group.",
+                       validator=validate_tracing_parameters,
+                       deprecate_info=c.deprecate(target='--disable_distributed_tracing', redirect='--disable_app_insights', hide=True))
+            c.argument('disable_app_insights',
+                       arg_type=get_three_state_flag(),
+                       help="Disable Application Insights, if not disabled and no existing Application Insights specified with --app-insights-key or --app-insights, will create a new Application Insights instance in the same resource group.",
+                       validator=validate_tracing_parameters)
+            c.argument('enable_java_agent', is_preview=True,
+                       arg_type=get_three_state_flag(),
```

Comment:
`false` would mean `disable` here?

Reply:
yes, false is disable. because this feature is in preview, the default value is disable